lib/onfleet/tasks.js

```diff
@@ -29,21 +53,34 @@ class Tasks {
     }
 
     /**
-     * Gets a list of all tasks that were scheduled for today. Failed tasks (or those
-     * with success=false) are included by default but can be excluded by providing an
-     * optional parameter.
-     *
-     * @param {boolean=} includeFailed
+     * Gets a list of all tasks that were scheduled for the same day as the given date.
+     * Failed tasks (or those with success=false) are included by default but can be excluded
+     * by providing an optional parameter.
+     * @param {Moment} date
+     * @param {boolean} includeFailed
      * @return {Promise}
      */
-    today(includeFailed = true) {
+    onDay(date, includeFailed=true) {
+
+        let from = date.clone().startOf('day');
+        let to = date.clone().endOf('day');
+
+        let tasks = [];
+        let thisTask = this;
+
+        function getAllPages(from, to, lastId) {
+            return thisTask.all(from, to, lastId)
+                .then(result => {
+                    tasks.push(...result.tasks);
```

Comment:
There's a possibility here that a task will be added twice if it is moved from page N to page N+1 during the call. Should wait and see if it happens though